frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/block_menu/block_menu_button.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:appflowy/generated/flowy_svgs.g.dart';
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
+import 'package:flowy_infra_ui/widget/flowy_tooltip.dart';
+import 'package:flutter/material.dart';
+
+class MenuBlockButton extends StatelessWidget {
+  const MenuBlockButton({
+    super.key,
+    required this.tooltip,
+    required this.iconData,
+    required this.onTap,
+  });
+
+  final VoidCallback onTap;
+  final String tooltip;
+  final FlowySvgData iconData;
+
+  @override
+  Widget build(BuildContext context) {
+    return FlowyButton(
+      useIntrinsicWidth: true,
+      onTap: onTap,
```

Comment:
Can make it nullable, then you don't have to pass `() {}` here https://github.com/AppFlowy-IO/AppFlowy/pull/4734/files#diff-fc379d4f0bf14d847cd3f81b8ce29be4fbf09893600d14c2fc60cfdbf0bccb06R152
```suggestion
    this.onTap,
  });

  final String tooltip;
  final FlowySvgData iconData;
  final VoidCallback? onTap;

  @override
  Widget build(BuildContext context) {
    return FlowyButton(
      useIntrinsicWidth: true,
      onTap: onTap,
```